src/ZwiftPacketMonitor.csproj

```diff
@@ -1,26 +1,30 @@
-<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework> <!-- netstandard2.0 -->
     <id>ZwiftPacketMonitor</id>
-    <Version>1.2.8</Version>
+    <Version>1.2.9.1</Version>
     <GeneratePackageOnBuild Condition="'$(Configuration)' == 'Release'">true</GeneratePackageOnBuild>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <Authors>Brad Walker (@braddwalker)</Authors>
+    <Authors>ruffk / Brad Walker (@braddwalker)</Authors>
     <Description>This is a port of the node zwift-packet-monitor project which monitors Zwift network packets</Description>
     <Title>Captures TCP and UDP packets produced/consumed by the Zwift online cycling simulator</Title>
-    <PackageProjectUrl>https://github.com/braddwalker/ZwiftPacketMonitor</PackageProjectUrl>
-    
+    <PackageProjectUrl>https://github.com/ruffk/ZwiftPacketMonitor</PackageProjectUrl>
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <!-- forward compatibility with expcted upcoming nuget changes https://github.com/NuGet/Home/wiki/Embedding-and-displaying-NuGet-READMEs -->
-    <PackageReadmeFile>../readme.md</PackageReadmeFile>
+    <PackageReadmeFile>readme.md</PackageReadmeFile>
+    <PackageId>ruffk.ZwiftPacketMonitor</PackageId>
+    <PackageTags>zwift, network, packet, protobuf</PackageTags>
+    <PackageReleaseNotes>Added support for SharpPcap 6.0 after breaking changes.  Changed TargetFramework to net5.0</PackageReleaseNotes>
+    <RepositoryUrl>https://github.com/ruffk/ZwiftPacketMonitor</RepositoryUrl>
```

Comment:
This URLs should reflect the correct project URL -- https://github.com/braddwalker/ZwiftPacketMonitor

Reply:
Done